tests/pocket_ic_test.py

```diff
@@ -58,7 +59,7 @@ def test_canister_exists_negative(self):
     def test_cycles_balance(self):
         canister_id = self.pic.create_canister()
         self.pic.add_cycles(canister_id, 6_666)
-        self.assertEqual(self.pic.get_cycles_balance(canister_id), 6_666)
+        self.assertEqual(self.pic.get_cycles_balance(canister_id)["cycles"], 6_666)
```

Comment:
Either we have to remove the type hint of the return type of `get_cycles_balance` (it's claiming it's returning an `int`), or we do the `["cycles"]` in there, which I think is preferable.